pkg/controllers/netpol/network_policy_controller.go

```diff
@@ -62,9 +64,8 @@ var (
 
 // NetworkPolicyController struct to hold information required by NetworkPolicyController
 type NetworkPolicyController struct {
-	nodeIP                  net.IP
 	nodeHostName            string
-	serviceClusterIPRange   net.IPNet
+	serviceClusterIPRanges  []net.IPNet
```

Comment:
This might be more work, but it might be beneficial to move to https://pkg.go.dev/net/netip#Prefix see also: https://tailscale.com/blog/netaddr-new-ip-type-for-go/

Reply:
Interesting... We may want to consider this one day, but for now it would be a pretty large change on top of an already crazy amount of change. So I think we should leave this PR with net.IP for now and later as the netaf library matures consider changing out across the whole project.